server/research/endpoints.js

```diff
@@ -1,22 +1,20 @@
 // For getting all interactions
 function interactionsEndpoint(pool, req, res) {
-  // query!!!
-  const sql = `
-    SELECT
-      id,
-      interaction,
-      timestampz,
-      session
-        
-    FROM interactions`;
+  const sql = `SELECT * FROM interactions`;
 
+  pool.query(sql)
+    .then(results => {
+      console.log(results.rows);
+      console.log(JSON.parse(JSON.stringify(results.rows)));
+      console.log('response ended');
+      res.set('Content-Type', 'application/json');
+      res.json(JSON.parse(JSON.stringify(results.rows)));
```

Comment:
Should this send a JS object or a string of JSON?

Reply:
From what I understand, res.json() needs a json object. 

results.rows gives us some oddly formatted object that is not json so by doing JSON.stringify(), I fix the formatting so that I can JSON.parse() it into a json object

Reply:
hmm, so `JSON.parse(JSON.stringify(whatever))` should have no impact and should just return whatever.

The issue might be that `result.rows` returns an array, which can't be a top-level JSON response.  I'm wondering if `res.json({ rows: results.rows })`might do the trick here?

Reply:
That worked! 👍 

I think what ended up happening with my code was that the JSON.stringify() stripped out the array stuff that was giving res.json() problems. It didn't occur to me I could also just get the bare json using `{ rows: results.rows }`

Reply:
Hmmmm. actually, after poking around with this some more, I think the original problem is still here. 

Doing `res.json({ rows: results.rows })` gives me a response like this:
<img width="338" alt="screen shot 2018-01-19 at 3 39 40 pm" src="https://user-images.githubusercontent.com/11862658/35170688-124f6cd0-fd2f-11e7-9237-58dbddb01de7.png">

Whereas `res.json(JSON.parse(JSON.stringify(results.rows)))` gives me something like this:
<img width="491" alt="screen shot 2018-01-19 at 3 39 53 pm" src="https://user-images.githubusercontent.com/11862658/35170729-34201f8a-fd2f-11e7-8509-c513a438972a.png">

I think the main issue is the `anonymous` at the beginning of each entry in the array. Is there a simpler way to cut that bit out?